app/javascript/home/Home.vue

```diff
@@ -77,6 +77,16 @@ function setScrollToFragmentTimeouts() {
 </script>
 
 <style lang="scss">
+:root {
+  --user-header-height: 32px;
+  --main-bg-color: var(--color-neutral-950);
+  --main-text-color: var(--color-neutral-100);
+}
+
+.logged-out {
+  --user-header-height: 0px;
+}
```

Comment:
This is a bit hacky, but it works to situate the logged-in header height and the navigation header as desired relative to each other, allowing for the fact that a user might not be logged in.